src/index.html

```diff
@@ -9,6 +9,33 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+
+    <header class="container">
+      <div class="containerLogo">
+        <a href="#home" class="linkLogo">
+          <img class="logo" src="../src/images/logo.png" 
+            alt="Store logo MOYO">
+        </a>
+      </div>
+
+      <nav class="navBlock">
+        <ul class="ulLink">
+          <li class="navList"><a href="#" class="is-active link">Apple</a></li>
```

Comment:
It's hard to read. Smth like this would be better
```suggestion
          <li class="navList">
            <a href="#" class="is-active link">Apple</a>
          </li>
```